templates/ec2/sc-ec2-linux-docker-notebook.yaml

```diff
@@ -348,17 +337,17 @@ Resources:
                 -e DOCKER_STACKS_JUPYTER_CMD=notebook \
                 -e SYNAPSE_TOKEN_AWS_SSM_PARAMETER_NAME=${SYNAPSE_TOKEN_AWS_SSM_PARAMETER_NAME} \
                 -e AWS_DEFAULT_REGION=${AWS_REGION} \
-                -v ${EC2_WORK_DIR}:${JUPYTER_WORK_DIR} \
+                -v ${WORK_VOLUME_NAME}:${JUPYTER_WORK_DIR} \
                 --network ${NETWORK_NAME} ${DOCKER_IMAGE} \
                 bash -c "start-notebook.py --IdentityProvider.token='' --NotebookApp.base_url=/${EC2_INSTANCE_ID} \
-                --notebook-dir=${JUPYTER_WORK_DIR}"
+                --ServerApp.root_dir=${JUPYTER_WORK_DIR}"
               env:
                 NETWORK_NAME:
                   'Fn::FindInMap': [GlobalVars, DockerNetworkName, Value]
                 NOTEBOOK_CONTAINER_NAME:
                   'Fn::FindInMap': [GlobalVars, NotebookContainerName, Value]
-                EC2_WORK_DIR:
-                  'Fn::FindInMap': [NotebookTypeToInitParams, !Ref NotebookType, EC2WorkDir]
+                WORK_VOLUME_NAME:
```

Comment:
(Optional) I would suggest normalizing JUPYTER_WORK_DIR and RSTUDIO_WORK_DIR to just `NOTEBOOK_WORK_DIR`